website/docs/r/nsxt_edgegateway_bgp_ip_prefix_list.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_edgegateway_bgp_ip_prefix_list"
+sidebar_current: "docs-vcd-resource-nsxt-edgegateway-bgp-ip-prefix-list"
+description: |-
+  Provides a resource to manage NSX-T Edge Gateway BGP IP Prefix Lists. IP prefix lists can contain 
+  single or multiple IP addresses and can be used to assign BGP neighbors with access permissions 
+  for route advertisement.
+---
+
+# vcd\_nsxt\_edgegateway\_bgp\_ip\_prefix\_list
+
+Supported in provider *v3.7+* and VCD 10.2+ with NSX-T
+
+Provides a resource to manage NSX-T Edge Gateway BGP IP Prefix Lists. IP prefix lists can contain
+single or multiple IP addresses and can be used to assign BGP neighbors with access permissions for
+route advertisement.
+
+
+## Example Usage (BGP IP Prefix List with multiple entries)
+
+```hcl
+resource "vcd_nsxt_edgegateway_bgp_ip_prefix_list" "testing" {
+  org = "cloud"
+
+  edge_gateway_id = data.vcd_nsxt_edgegateway.testing.id
+
+  name        = "sample-ip-prefix-list"
+  description = "This definition is meant only to demostrate capabilities"
+
+  ip_prefix {
+    network = "10.10.10.0/24"
+    action  = "PERMIT"
+  }
+
+  ip_prefix {
+    network = "20.10.10.0/24"
+    action  = "DENY"
+  }
+
+  ip_prefix {
+    network = "2001:db8::/48"
+    action  = "DENY"
+  }
+
+  ip_prefix {
+    network                  = "30.10.10.0/24"
+    action                   = "DENY"
+    greater_than_or_equal_to = "25"
+    less_than_or_equal_to    = "27"
+  }
+
+  ip_prefix {
+    network                  = "40.0.0.0/8"
+    action                   = "PERMIT"
+    greater_than_or_equal_to = "16"
+    less_than_or_equal_to    = "24"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful
+  when connected as sysadmin working across different organisations
+* `edge_gateway_id` - (Required) The ID of the edge gateway (NSX-T only). Can be looked up using
```

Comment:
Let's capitalize Edge Gateway as in other places.

Reply:
Done. Found 11 occurrences in markdown docs and replaced all of them.